kinto/plugins/history/listener.py

```diff
@@ -26,7 +27,7 @@ def on_resource_changed(event):
 
     targets = []
     for impacted in event.impacted_records:
-        target = impacted['new']
+        target = impacted['old'] if action == 'delete' else impacted['new']
```

Comment:
So we don't have that previously available `deleted` attribute from the tombstone available... should we keep it ? yeah that's super weird